Rakefile

```diff
@@ -3,142 +3,114 @@ require 'erb'
 require 'net/http'
 require 'tmpdir'
 
-def fetch(uri, limit = 10)
-  # puts "Fetching #{uri}" # Uncomment to debug what URI's are being fetched
-  raise 'too many HTTP redirects' if limit == 0
-  response = Net::HTTP.get_response(URI(uri))
-  case response
-  when Net::HTTPSuccess then
-    response
-  when Net::HTTPRedirection then
-    fetch(response['location'], limit - 1)
-  else
-    raise "Request for listing failed: #{response.body}"
-  end
-end
-
+# URL where our MacOS builds are located
+MACOS_BASE_URL = 'https://downloads.voxpupuli.org/mac'
+# MacOS architectures we can support (symbolized)
+SUPPORTED_ARCHITECTURES = %i[arm64 x86_64]
+# MacOS versions we can support
+SUPPORTED_OS_VERSIONS = %w[13 14 15]
```

Comment:
Have you tried running our 15 agent against 13/14? I imagine it might work, but we don't have ones built for those versions specifically.

Reply:
Let me try to run it in a tart v13/v14 VMs.. it won't catch any signing issue, but at least we can check if it works at all.

Reply:
Just tried in a v14 tart VM via CLI-only. `hdiutil` and `installer` are happy, `puppet --version` works fine. Not sure if SIP will complain on real hardware though..